src/_output/vue/app.vue

```diff
@@ -26,20 +26,27 @@
                       @change="checkSentenceCount"
                       :id="itemIdWithValue('sentenceCount')"
                   >
-                  <label :for="itemIdWithValue('sentenceCount')">sentences around</label>
+                  <label :for="itemIdWithValue('sentenceCount')" >sentences around</label>
                 </span>
             </p>
 
-            <al-groups-view-full :full-data="$parent.fullData" v-if="viewType === 'viewFull'" />
-            <al-groups-view-columns :full-data="$parent.fullData" v-if="viewType === 'view3Columns'" />
-            <al-groups-view-short :full-data="$parent.fullData" v-if="viewType === 'viewShort'" />
-            <al-groups-view-sentence :full-data="$parent.fullData" :sentence-count = "sentenceCount" v-if="viewType === 'viewSentence'" />
-            <al-groups-view-equivalence :full-data="$parent.fullData" v-if="viewType === 'viewEquivalence'" />
+            <languages-block :fullData="fullData" v-if="languagesList.length > 1"
+                @changeLanguageOrder = "changeLanguageOrder" @updateVisibility = "updateVisibility"/>
+
+            <al-groups-view-full :fullData="fullData" :languageTargetIds = "languageTargetIds" v-if="viewType === 'viewFull'" />
+            <al-groups-view-columns :fullData="fullData" :languageTargetIds = "languageTargetIds"  v-if="viewType === 'view3Columns'" />
+            <al-groups-view-short :fullData="fullData" :languageTargetIds = "languageTargetIds"  v-if="viewType === 'viewShort'" />
+            <al-groups-view-sentence :fullData="fullData" :languageTargetIds = "languageTargetIds"  :sentence-count = "sentenceCount" v-if="viewType === 'viewSentence'" />
+            <al-groups-view-equivalence :fullData="fullData" :languageTargetIds = "languageTargetIds"  v-if="viewType === 'viewEquivalence'" />

```

Comment:
This is the main component `App` - where are:
- all display views
- editor's tabs
- languages block